install_files/securedrop-app-code/DEBIAN/postinst

```diff
@@ -84,6 +84,9 @@ case "$1" in
       chmod 0700 "$dir"
     done
 
+    # Ensure required gpg-agent.conf is in place, see #4013
+    echo allow-loopback-pinentry > /var/lib/securedrop/keys/gpg-agent.conf
+
```

Comment:
Personally I think this should be something like

```bash
if ! grep -qE '^allow-loopback-pinentry$' /var/lib/securedrop/keys/gpg-agent.conf; then
    echo allow-loopback-pinentry >> /var/lib/securedrop/keys/gpg-agent.conf
fi
```

just to make this a bit more future-proof. But this is fine to merge as is imo because CI will catch this.